starfish/imagestack/imagestack.py

```diff
@@ -122,7 +126,12 @@ def __init__(
         }
         self._tile_shape = tile_shape
         self._tile_data = tile_data
-        self._log: List[dict] = list()
+
+        # check for existing log info
+        if STARFISH in tile_data.extras:
+            self._log = loads(tile_data.extras[STARFISH])[LOG]
```

Comment:
To make it 100% crystal clear that this is a starfish-ism, let's add something to the ImageStack parsers such that `TileCollectionData` explicitly provides controlled vocabulary metadata.

That might look like:
```
class StarfishMetadata:
    def log(self) -> List[dict]:
        ....

class TileCollectionData:
    def get_starfish_metadata(self) -> StarfishMetadata:
        raise NotImplementedError

class TileSetData:
    def get_starfish_metadata(self) -> StarfishMetadata:
        return json.loads(self.extras[STARFISH])
```